exercises/practice/camicia/test/camicia_test.exs

```diff
@@ -0,0 +1,1432 @@
+defmodule CamiciaTest do
+  use ExUnit.Case
+
+  # @tag :pending
+  test "two cards, one trick" do
+    playerA = ["2"]
+    playerB = ["3"]
+    expected = {:finished, 2, 1}
+    assert Camicia.simulate(playerA, playerB) == expected
+  end
+
+  @tag :pending
+  test "three cards, one trick" do
+    playerA = ["2", "4"]
+    playerB = ["3"]
+    expected = {:finished, 3, 1}
+    assert Camicia.simulate(playerA, playerB) == expected
+  end
+
+  @tag :pending
+  test "four cards, one trick" do
+    playerA = ["2", "4"]
+    playerB = ["3", "5", "6"]
+    expected = {:finished, 4, 1}
+    assert Camicia.simulate(playerA, playerB) == expected
+  end
+
+  @tag :pending
+  test "the ace reigns supreme" do
+    playerA = ["2", "A"]
+    playerB = ["3", "4", "5", "6", "7"]
+    expected = {:finished, 7, 1}
+    assert Camicia.simulate(playerA, playerB) == expected
+  end
+
+  @tag :pending
+  test "the king beats ace" do
+    playerA = ["2", "A"]
+    playerB = ["3", "4", "5", "6", "K"]
+    expected = {:finished, 7, 1}
+    assert Camicia.simulate(playerA, playerB) == expected
+  end
+
+  @tag :pending
+  test "jack beats king" do
```

Comment:
For easier maintenance, the test names should be identical to those in problem specs (https://github.com/exercism/problem-specifications/blob/main/exercises/camicia/canonical-data.json#L74-L87)

```suggestion
  test "the queen seduces the king" do
```